src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/MFD/AS1000_MFD.html

```diff
@@ -1,10 +1,4 @@
-﻿<script>
-    Include.addScript("/JS/debug.js", function () {
-        g_modDebugMgr.AddConsole(null);
-    });
-</script>
-
-<script type="text/html" id="AS1000_MFD">
+﻿<script type="text/html" id="AS1000_MFD">
```

Comment:
Where do we sit on this regarding if you don't have the dev kit? Was it determined that breaks it?

Reply:
Accidentally selected the wrong bit of code, it's the debug script inclusion